main.go

```diff
@@ -197,6 +197,32 @@ func findFailedTests(dirName string, env map[string]string, threshold int) ([]te
 	return failedTests, nil
 }
 
+func addFailedTest(failedTests []testCase, tc junit.Test, env map[string]string) []testCase {
+	if !isSubTest(tc) {
+		return append(failedTests, NewTestCase(tc, env))
+	}
+	return addSubTestToFailedTest(tc, failedTests, env)
+}
+
+func isSubTest(tc junit.Test) bool {
+	return strings.Contains(tc.Name, "/")
```

Comment:
Although this condition is true for go subtests it's not enough as it will merge tests from e2e that contains `/` in their names (see https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/4254/pull-ci-stackrox-stackrox-master-gke-postgres-qa-e2e-tests/1609808665097801728) 
How about checking if test name is prefixed with previous test name or at least if test is go by checking classname to contains golang package.

Reply:
Added the check to ensure we only attempt to group go tests based on the go module's prefix.